src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -84,8 +84,11 @@ public partial class JsonArray : System.Text.Json.JsonNode, System.Collections.G
         public void Insert(int index, uint item) { }
         [System.CLSCompliantAttribute(false)]
         public void Insert(int index, ulong item) { }
+        public int LastIndexOf(System.Text.Json.JsonNode item) { throw null; }
```

Comment:
Is this method just so that we implement a specific interface Or why do we need it?

Reply:
I looked though `ArrayList` for methods that might be useful (like `Sort`) and considered this one such, as well as consistent with `IndexOf`.